_data/software/zint-barcode-generator.yml

```diff
@@ -0,0 +1,36 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: A barcode encoding library supporting over 50 symbologies.
+    fr: >-
+      Une bibliothèque d'encodage de codes à barres prenant en charge plus de 50
+      symbologies.
+category: '43233400'
+homepageURL:
+  en: 'https://zint.github.io/'
```

Comment:
The readme lists http://www.zint.org.uk/ as the site, and the GH hasn't been updated since 2013